pkg/scheduler/redis_scheduler.go

```diff
@@ -158,6 +228,11 @@ func (s *RedisScheduler) Add(t Trigger) error {
 func (s *RedisScheduler) addToRedis(t Trigger, prev time.Time) error {
 	var timestamp time.Time
 	switch t := t.(type) {
+	case *EventTrigger:
+		hKey := eventsKey(t.Infos().Domain)
+		hMap := make(map[string]interface{})
+		hMap[t.ID()] = t.Infos().Arguments
+		return s.client.HMSet(hKey, hMap).Err()
```

Comment:
Indeed, did not see this method, thanks. Fixed ea44ec272657c16f388c1efd35d6c7b096ee180b